oci/private/pull.bzl

```diff
@@ -80,42 +80,63 @@ def _digest_into_blob_path(digest):
     digest_path = digest.replace(":", "/", 1)
     return "blobs/{}".format(digest_path)
 
-def _download(rctx, authn, identifier, output, resource, headers = {}, allow_fail = False):
+def _download(rctx, authn, identifiers, outputs, resource, headers = {}, allow_fail = False):
```

Comment:
i don't think the place to implement this, and i don't think we care about downloading manifests and small blobs like config concurrently. 

Can we just make download accept additional **kwargs and download only layers in parallel in the for loop below?

Reply:
Which kwargs would you have it accept?

Reply:
accepting an additional `block` argument and applying it conditionally the same way as the headers argument would be fine here. 

then down below where we download layers sequentially can pass `block` argument unconditionally and collect results and do `[r.wait() for r in results if "wait" in r]`. that would be cleanest implementation imho.